.github/workflows/ci-pr-validation.yml

```diff
@@ -90,8 +90,8 @@ jobs:
       - name: Build Node binaries lib
         run: |
           yarn install --ignore-scripts
```

Comment:
Could you verify both `yarn install` and `npm install`?

```bash
echo "Verifying yarn install"
yarn install --ignore-scripts
# Maybe removing the `node_modules` directory is unnecessary
rm -rf node_modules/
echo "Verifying npm install"
npm install --ignore-scripts
```

Reply:
Since we only cache for `npm` above:

```
cache: 'npm'
```

I don't think we should update the pkg manager here. Any motivation to make this change? If so, it can hide some subtle bugs.

Reply:
> Could you verify both yarn install and npm install?

In fact, used `npm install` when building Windows as well as Linux Arch. This has verified that both yarn and npm can be used.

Reply:
> Any motivation to make this change? 

@tisonkun Because there is a weird issue that `yarn install` failed for the current master while `npm install` succeeded.